src/Raven.Studio/typescript/viewmodels/database/settings/databaseSettings.ts

```diff
@@ -0,0 +1,444 @@
+        this.filterKeys.throttle(500).subscribe(() => this.computeEntriesToShow());
+
+        this.isAnyMatchingEntries = ko.pureComputed(() => !!this.allEntries().filter(x => x.showEntry()).length);
+    }
+    
+    private computeEntriesToShow() {
+        this.allEntries().forEach(entry => entry.showEntry(this.shouldShowEntry(entry)));
+        
+        if (this.viewMode() === "summaryMode") {
+            this.summaryGridController().reset(false);
+        }
+    }
+
+    private shouldShowEntry(entry: models.settingsEntry) {
+
+        const categoryCondition =  (this.viewMode() === "editMode" && this.selectedCategory() === entry.data.Metadata.Category) ||
+                                    this.viewMode() === "summaryMode";
+
+        const filterCondition = this.entryContainsFilterText(entry);
+
+        const dropDownCondition = this.groupToShow() === "allEntries" ||
+                                 (this.groupToShow() === "databaseEntries" && !entry.isServerWideOnlyEntry()) ||
+                                 (this.groupToShow() === "customizedDatabaseEntries" && entry instanceof models.databaseEntry && entry.override());
+
+        return categoryCondition && filterCondition && dropDownCondition;
+    }
+    
+    private entryContainsFilterText(entry: models.settingsEntry) {
+        const searchText = this.filterKeys().toLocaleLowerCase();
+        return entry.keyName().toLocaleLowerCase().includes(searchText);
+    }
+
+    canActivate(args: any) {
+        return $.when<any>(super.canActivate(args))
+            .then(() => {
+                const deferred = $.Deferred<canActivateResultDto>();
+
+                this.isForbidden(!accessManager.default.operatorAndAbove());
+
+                if (this.isForbidden()) {
+                    deferred.resolve({can: true});
+                } else {
+                    const db: database = this.activeDatabase();
+                    this.fetchDatabaseSettings(db)
+                        .done(() => deferred.resolve({can: true}))
+                        .fail((response: JQueryXHR) => {
+                            deferred.resolve({redirect: appUrl.forStatus(db)});
+                        });
+                }
+
+                return deferred;
+            });
+    }
+
+    compositionComplete() {
+        super.compositionComplete();
+        
+        this.initSummaryGrid();
+        
+        this.processAfterFetch();
+        
+        $('#database-settings-view [data-toggle="tooltip"]').tooltip();
+        
+        $('.description').on("mouseenter", event => {
+           const target = $(event.currentTarget);
+           const targetId = event.currentTarget.id;
+
+           if (!target.data('bs.popover')) {
+               popoverUtils.longWithHover(target, {
+                   content: `<div class="description-text"><strong>${targetId}</strong><small>${this.getDescriptionFor(targetId)}</small></div>`,
```

Comment:
escape

Reply:
I think I don't need to duplicate, 
there is such an escape on the entry's `descriptionHtml` computed

Reply:
ok